pkg/render/intrusion_detection.go

```diff
@@ -54,12 +55,20 @@ const (
 	IntrusionDetectionInstallerJobName = "intrusion-detection-es-job-installer"
 	IntrusionDetectionControllerName   = "intrusion-detection-controller"
 
-	ADJobPodTemplateBaseName = "tigera.io.detectors"
-
+	ADJobPodTemplateBaseName     = "tigera.io.detectors"
```

Comment:
This field is used a s service account name. We don't typically use periods in service account names.

Reply:
https://github.com/tigera/operator/pull/1931/commits/136b82e64ba9d34bee78baa945ee24909eb56e53 fixed here